SparkPost/2web2ui
# FE-1169 - Configure Terser to drop console methods from production builds

[FE-1169](https://sparkpost.atlassian.net/browse/FE-1169)

### What Changed
- Upgraded `TerserPlugin` configuration to drop `console` methods from production builds
- See API reference here

### How To Test
- Checkout this branch and run `npm run build`, then inside the `./build` directory search for "Use the `ButtonLink`'" - you should not get any hits on the `main.<hash>.chunk.js` file for a `console.warn()` containing this content
- Checkout the `master` branch and repeat the above - you should get a hit for a `console.warn` inside the `main.<hash>.chunk.js` file


### To Do
